static-site/src/components/ListResources/Modal.tsx

```diff
@@ -125,8 +132,8 @@ const Title = styled.div`
 
 function _snapshotSummary(dates) {
```

Comment:
Might make some of the need for subsequent casts go away?

```suggestion
function _snapshotSummary(dates: [number]) {
```

Reply:
Updated to:

https://github.com/nextstrain/nextstrain.org/blob/213995c9b8a189285ec62094009204e0d981f70d/static-site/src/components/ListResources/Modal.tsx#L133-L137